### PR TITLE
Cursor highlight verticle and horizontal keymap

### PR DESCRIPTION
**Pull request by rocknrollMarc**

### CODE REVIEW

**Comment by alanrrb** [comment]
You've pushed your stuffs, take look at .gitconfig (and others files)

**Comment by rocknrollMarc** [comment]
Sorry guys my internet connection is really slow so I am having trouble connecting.

Very nice Keymaps and function to turn on line highlight also verticle Ill add a screensho

**Comment by rocknrollMarc** [comment]
Sorry guys my internet connection is really slow so I am having trouble connecting.

Very nice Keymaps and function to turn on line highlight also verticle Ill add a screensho

**Comment by rocknrollMarc** [comment]
Sorry Im not that firm in doing this pull request stuff jet ill try to fix it. I didnt want to push my stuff only the keymap file with the keymaps for the cursor highlight :)

**Comment by alanrrb** [comment]
Try open a new branch and `git rebase -i` on it to choose which commit you want to push.

File: deploy.zsh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/zsh
```
Comment (ianks) [inline deploy.zsh]: Can you rename this script to `getshitdone.sh` ?

**Pull request closed without merging**